WordPress/gutenberg
# Testing: Remove snapshot from Post Editor Template Mode test and target expected string instead

## What?


Remove the snapshot from the `Allow creating custom block templates in classic themes` test in the Post Editor Template Mode playwright tests.

## Why?


While working on #40875, I noticed that whenever I made a change to the classnames that are attached to the group block, this test would fail due to the snapshot test used, which looks at all the markup of the rendered preview, and if any markup has changed, the test fails.

I _think_, much like the test directly above it, that this test is just wanting to check that the added template is included in the preview. Testing that all markup is identical seems to be a bit outside the scope of this test, and could be making it a little more rigid than needed. So, in this PR, we switch to looking for the expected string within the response.

## How?


* Remove snapshot test
* Check for explicit text string `Just a random paragraph added to the template` instead

## Testing Instructions





Check that the playwright tests pass. You can also attempt to run it locally via:

```
npm run test-e2e:playwright -- test/e2e/specs/editor/various/post-editor-template-mode.spec.js
```
